docs/glossary.md

```diff
@@ -30,4 +30,4 @@
 + **Productions server**: A server on AppSpot. Can be the live server or a staging server.
 + **Spec**: System Specification.
 + **Staging green**: All test are passing against you own deployment on the GAE.
-+ **Staging server**: The server instance you created on AppSpot.
\ No newline at end of file
++ **Staging server**: The server instance you created on AppSpot.
```

Comment:
@wkurniawan07 I haven't change anything in that line. Actual both versions (Change and without change) of this file look same to me.